### ISSUE TITLE
UIWebView Deprecated

### ISSUE THREAD

**Issue by npalethorpe** [wirecard/paymentSDK-iOS#35]
I'm currently trying to remove all instances to the soon to be deprecated UIWebView and I've noticed the paymentSDK seems to make reference to the UIWebView in the `WDeCom` binary file found at `Pods/paymentSDK/eCom/WDeCom.framework/WDeCom`.

I'm running the latest 4.1.0 version. New app submissions with UIWebView references will be rejected from the end of this month (30th April) so its something of an urgent requirement.

Thanks,

**Comment by npalethorpe** [comment]
Hi Peter,

I've checked and there doesn't appear to be an issue with the AFNetworking pods, I believe the issue is related to the `paymentSDK/Scanner` which appears to use a dependancy of `GoogleMobileVision/TextDetector (~> 1.6.0)` which it seems also has a reference to UIWebView.

I'm currently piecing together ambiguous puzzle pieces so I apologise if I'm wrong!
  
If I do a `grep -r UIWebView ./*` in my project directory I get the following:
  
```
Binary file ./ios/Pods/paymentSDK/eCom/WDeCom.framework/WDeCom matches
Binary file ./ios/Pods/GoogleMobileVision/Detector/Frameworks/GoogleMobileVision.framework/GoogleMobileVision matches
```

Can you replicate? I'm making an assumption that this is the cause to me still receiving the deprecation email from Apple due to all other search results (from the above grep) being comments.

Thanks,

**Comment by PeterStavny** [comment]
Hi Neil,

Yes I replaced it. 

Binary file ./ios/Pods/paymentSDK/Com/WDeCom.framework/WDeCom matches - contains not executable forgotten text "UIWebView". It should not be problem. 
As you mentioned problem is GoogleMobileVision. 
We are not able to avoid this library or avoid only the specific subspec. But this framework is provided by the WdeComScanner framework only. 
To avoid the "UIWebView" issue I recommend you avoid modules with the WdeComScanner framework dependecy for now - let say a "quick solution". 

You are able to do it by podfile change (use instead pod 'paymentSDK'):
    pod 'paymentSDK/Core', '~> 4.1.0'
    pod 'paymentSDK/CoreCard', '~> 4.1.0'
    pod 'paymentSDK/Alipay', '~> 4.1.0'
    pod 'paymentSDK/ApplePay', '~> 4.1.0'
    pod 'paymentSDK/Card', '~> 4.1.0'
    pod 'paymentSDK/PayPal', '~> 4.1.0'
    pod 'paymentSDK/SEPA', '~> 4.1.0'
    pod 'paymentSDK/ZAPP', '~> 4.1.0'

If you use classes with WDEC.......ScannerGallery in your code you shall remove the postfix ScannerGallery. You will lost the scanner functonality of course.

Our and my apologise for your problems.

Please let me know the result.
Thanks,

**Comment by npalethorpe** [comment]
Hi Peter,

Thanks for your help - I've replaced the pod as you've mentioned and that does appear to have resolved the issue and I'm not longer getting a deprecated UIWebView email!

I'll keep an eye out for when the scanning functionality gets added back in.

Many thanks,

**Comment by Gray-Wind** [comment]
I have the same issue, but I cannot update AFNetworking for some reason:

```
[!] CocoaPods could not find compatible versions for pod "AFNetworking/Serialization":
  In Podfile:
    paymentSDK/Core (~> 4.1.0) was resolved to 4.1.0, which depends on
      AFNetworking/Serialization (~> 3.2.1)
```

**Comment by PeterStavny** [comment]
Please follow #37
